homeassistant/components/zha/core/const.py

```diff
@@ -157,20 +157,23 @@ class RadioType(enum.Enum):
     """Possible options for radio type."""
 
     ezsp = (
-        "ESZP: HUSBZB-1, Elelabs, Telegesis, Silabs EmberZNet protocol",
+        "EZSP = Silicon Labs EmberZNet protocol (ex; Elelabs, HUSBZB-1, Telegesis)",
         bellows.zigbee.application.ControllerApplication,
     )
     deconz = (
-        "Conbee, Conbee II, RaspBee radios from dresden elektronik",
+        "deCONZ = dresden elektronik deCONZ protocol (ex; ConBee I/II, RaspBee I/II)",
```

Comment:
There are only ConBees and RaspBees, so for this one could remove ex; completely
```suggestion
        "deCONZ = dresden elektronik deCONZ protocol: ConBee I/II, RaspBee I/II",
```